spinalcordtoolbox/deepseg/models.py

```diff
@@ -76,6 +76,14 @@
         "description": "Grey/white matter seg on exvivo human T2w.",
         "contrasts": ["t2"],
         "default": False,
+    },
+    "model_7T_multiclass_gm_sc_unet2d": {
+        "url": [
+            "https://github.com/ivadomed/model_seg_gm-wm_t2star_7T_unet3d-multiclass/blob/main/Model_SC_GM_7T.zip"
+        ],
+        "description": "SC/GM multiclass segmentation at 7T t2star. Model from CRMBM-CEMEREM Aix-Marseille University",
```

Comment:
```suggestion
        "description": "SC/GM multiclass segmentation on T2*-w contrast at 7T. The model was created by Nilser Laines Medina at CRMBM-CEMEREM Aix-Marseille University",
```
Important to have a contact person